cypress/integration/Home/index.spec.ts

```diff
@@ -1,13 +1,9 @@
 describe('Home page', () => {
-  const scrollSurveyItem = (times = 1) => {
-    for (let time = 1; time <= times; time++) {
-      cy.get('[data-testid="survey-list"]')
-        .trigger('pointerdown')
-        .trigger('pointermove', 'right')
-        .trigger('pointermove', 'left')
-        .trigger('pointerup');
-      cy.wait(150);
-    }
+  const goToSurveyItem = (index: number) => {
+    cy.get('[data-testid="survey-list"]')
+      .get('.swiper-pagination-bullet')
+      .eq(index - 1)
+      .click();
```

Comment:
Should we keep 1 scroll item so that we can cover both cases (click to switch AND scroll to switch)? 🤔

Reply:
Updated in [f3f0cb2](https://github.com/hanam1ni/react-surveyor/pull/81/commits/f3f0cb27f531c1cdc0f2be34eefe48a0a6a2cb7a)